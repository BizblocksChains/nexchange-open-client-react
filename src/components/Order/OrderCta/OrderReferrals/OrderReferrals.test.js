import React from 'react';
import { shallow } from 'enzyme';
import OrderReferrals from './OrderReferrals';
import order from 'Mocks/order';

describe('OrderReferrals', () => {
  it('renders correctly', () => {
    const wrapShallow = shallow(<OrderReferrals order={order} />).dive();
    expect(wrapShallow).toMatchSnapshot();
  });
});
