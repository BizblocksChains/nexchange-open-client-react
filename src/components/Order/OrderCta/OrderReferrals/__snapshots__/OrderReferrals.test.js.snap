// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderReferrals renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <I18n
    ns="translations"
>
    [Function]
</I18n>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="box container"
>
          <div
                    className="row"
          >
                    <div
                              className="col-xs-12 visible-xs text-center"
                    >
                              <img
                                        alt="notify.alt"
                                        className="img"
                                        src="man.png"
                              />
                    </div>
                    <div
                              className="col-xs-12 col-sm-7 text"
                    >
                              <h2
                                        className="title"
                              >
                                        <Interpolate
                                                  className=""
                                                  i18nKey="referral.affiliate2"
                                                  selectedCoin="BitDegree"
                                        />
                              </h2>
                              <I18n
                                        ns="translations"
                              >
                                        [Function]
                              </I18n>
                    </div>
                    <div
                              className="col-ms-2 col-sm-5 hidden-xs text-center"
                    >
                              <img
                                        alt="notify.alt"
                                        className="img"
                                        src="man.png"
                              />
                    </div>
          </div>
</div>,
        <OrderReferralTerms
          onClose={[Function]}
          show={false}
/>,
      ],
      "className": "col-xs-12",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="row"
>
            <div
                        className="col-xs-12 visible-xs text-center"
            >
                        <img
                                    alt="notify.alt"
                                    className="img"
                                    src="man.png"
                        />
            </div>
            <div
                        className="col-xs-12 col-sm-7 text"
            >
                        <h2
                                    className="title"
                        >
                                    <Interpolate
                                                className=""
                                                i18nKey="referral.affiliate2"
                                                selectedCoin="BitDegree"
                                    />
                        </h2>
                        <I18n
                                    ns="translations"
                        >
                                    [Function]
                        </I18n>
            </div>
            <div
                        className="col-ms-2 col-sm-5 hidden-xs text-center"
            >
                        <img
                                    alt="notify.alt"
                                    className="img"
                                    src="man.png"
                        />
            </div>
</div>,
          "className": "box container",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="col-xs-12 visible-xs text-center"
>
                <img
                                alt="notify.alt"
                                className="img"
                                src="man.png"
                />
</div>,
              <div
                className="col-xs-12 col-sm-7 text"
>
                <h2
                                className="title"
                >
                                <Interpolate
                                                className=""
                                                i18nKey="referral.affiliate2"
                                                selectedCoin="BitDegree"
                                />
                </h2>
                <I18n
                                ns="translations"
                >
                                [Function]
                </I18n>
</div>,
              <div
                className="col-ms-2 col-sm-5 hidden-xs text-center"
>
                <img
                                alt="notify.alt"
                                className="img"
                                src="man.png"
                />
</div>,
            ],
            "className": "row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="notify.alt"
                  className="img"
                  src="man.png"
/>,
                "className": "col-xs-12 visible-xs text-center",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "notify.alt",
                  "className": "img",
                  "src": "man.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2
                    className="title"
>
                    <Interpolate
                                        className=""
                                        i18nKey="referral.affiliate2"
                                        selectedCoin="BitDegree"
                    />
</h2>,
                  <I18n
                    ns="translations"
>
                    [Function]
</I18n>,
                ],
                "className": "col-xs-12 col-sm-7 text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": <Interpolate
                      className=""
                      i18nKey="referral.affiliate2"
                      selectedCoin="BitDegree"
/>,
                    "className": "title",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "className": "",
                      "i18nKey": "referral.affiliate2",
                      "selectedCoin": "BitDegree",
                    },
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {
                    "children": [Function],
                    "ns": "translations",
                  },
                  "ref": null,
                  "rendered": [Function],
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="notify.alt"
                  className="img"
                  src="man.png"
/>,
                "className": "col-ms-2 col-sm-5 hidden-xs text-center",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "notify.alt",
                  "className": "img",
                  "src": "man.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "onClose": [Function],
          "show": false,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="box container"
>
            <div
                        className="row"
            >
                        <div
                                    className="col-xs-12 visible-xs text-center"
                        >
                                    <img
                                                alt="notify.alt"
                                                className="img"
                                                src="man.png"
                                    />
                        </div>
                        <div
                                    className="col-xs-12 col-sm-7 text"
                        >
                                    <h2
                                                className="title"
                                    >
                                                <Interpolate
                                                            className=""
                                                            i18nKey="referral.affiliate2"
                                                            selectedCoin="BitDegree"
                                                />
                                    </h2>
                                    <I18n
                                                ns="translations"
                                    >
                                                [Function]
                                    </I18n>
                        </div>
                        <div
                                    className="col-ms-2 col-sm-5 hidden-xs text-center"
                        >
                                    <img
                                                alt="notify.alt"
                                                className="img"
                                                src="man.png"
                                    />
                        </div>
            </div>
</div>,
          <OrderReferralTerms
            onClose={[Function]}
            show={false}
/>,
        ],
        "className": "col-xs-12",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="row"
>
              <div
                            className="col-xs-12 visible-xs text-center"
              >
                            <img
                                          alt="notify.alt"
                                          className="img"
                                          src="man.png"
                            />
              </div>
              <div
                            className="col-xs-12 col-sm-7 text"
              >
                            <h2
                                          className="title"
                            >
                                          <Interpolate
                                                        className=""
                                                        i18nKey="referral.affiliate2"
                                                        selectedCoin="BitDegree"
                                          />
                            </h2>
                            <I18n
                                          ns="translations"
                            >
                                          [Function]
                            </I18n>
              </div>
              <div
                            className="col-ms-2 col-sm-5 hidden-xs text-center"
              >
                            <img
                                          alt="notify.alt"
                                          className="img"
                                          src="man.png"
                            />
              </div>
</div>,
            "className": "box container",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div
                  className="col-xs-12 visible-xs text-center"
>
                  <img
                                    alt="notify.alt"
                                    className="img"
                                    src="man.png"
                  />
</div>,
                <div
                  className="col-xs-12 col-sm-7 text"
>
                  <h2
                                    className="title"
                  >
                                    <Interpolate
                                                      className=""
                                                      i18nKey="referral.affiliate2"
                                                      selectedCoin="BitDegree"
                                    />
                  </h2>
                  <I18n
                                    ns="translations"
                  >
                                    [Function]
                  </I18n>
</div>,
                <div
                  className="col-ms-2 col-sm-5 hidden-xs text-center"
>
                  <img
                                    alt="notify.alt"
                                    className="img"
                                    src="man.png"
                  />
</div>,
              ],
              "className": "row",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    alt="notify.alt"
                    className="img"
                    src="man.png"
/>,
                  "className": "col-xs-12 visible-xs text-center",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "notify.alt",
                    "className": "img",
                    "src": "man.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <h2
                      className="title"
>
                      <Interpolate
                                            className=""
                                            i18nKey="referral.affiliate2"
                                            selectedCoin="BitDegree"
                      />
</h2>,
                    <I18n
                      ns="translations"
>
                      [Function]
</I18n>,
                  ],
                  "className": "col-xs-12 col-sm-7 text",
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": <Interpolate
                        className=""
                        i18nKey="referral.affiliate2"
                        selectedCoin="BitDegree"
/>,
                      "className": "title",
                    },
                    "ref": null,
                    "rendered": Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "class",
                      "props": Object {
                        "className": "",
                        "i18nKey": "referral.affiliate2",
                        "selectedCoin": "BitDegree",
                      },
                      "ref": null,
                      "rendered": null,
                      "type": [Function],
                    },
                    "type": "h2",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {
                      "children": [Function],
                      "ns": "translations",
                    },
                    "ref": null,
                    "rendered": [Function],
                    "type": [Function],
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": <img
                    alt="notify.alt"
                    className="img"
                    src="man.png"
/>,
                  "className": "col-ms-2 col-sm-5 hidden-xs text-center",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "alt": "notify.alt",
                    "className": "img",
                    "src": "man.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": "img",
                },
                "type": "div",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "onClose": [Function],
            "show": false,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
